### ISSUE TITLE
Confused by meaning of sentence "how to complete ... individual form controls"

### ISSUE THREAD

**Issue by happywebdiva** [w3c/wai-tutorials#204]
On the Form Instructions page (http://w3c.github.io/wai-tutorials/forms/instructions/), I get confused by the first sentence: "Provide instructions to help users understand how to complete the form and individual form controls."  It's that last phase, i.e., "how to complete ... individual form controls".  I think that technically it's correct, but when I read this I first think, huh -- how can you fill out the control type, e.g. a button?  Would it maybe be more clear to say "individual form elements"?

**Comment by yatil** [comment]
What about form **fields**? I know that Shadi @nitedog changed *form elements* to *form controls* almost anywhere, but I don’t know the exact reasons.

**Comment by nitedog** [comment]
Rationale for background: I tried to use controls throughout, as it is most generic. Fields are a type of controls that are used for text input. Elements is the HTML reflection, but sometimes controls are composed of several elements. I kept "form fields" and "form elements" on the few occasions where we specifically meant input fields or HTML form elements.

Having said that, I agree with @happywebdiva on this comment - "complete ... individual form controls" does not make sense. How about removing "and individual form controls" all together? Or change it to "and _use_ the individual form controls"? Or change back to "form fields", though sometimes the instructions are also for non-fields (eg. which options to select, like "check 'new customer' to create a new account" or such). The latter could be seen as marginal, though. I can live with "form fields" in this case, if needed.
